scripts/get_server_cert.sh

```diff
@@ -0,0 +1,9 @@
+ENDPOINT_ADDRESS=$1
```

Comment:
`#!/bin/bash` missing

Reply:
two identical files with one grep difference - can we extract common logic to separate file - `get_cert.sh`